webpack/assets/javascripts/spice.js

```diff
@@ -57,10 +56,7 @@ function spiceError(e) {
 
 function spiceSuccess(m) {
   $('#spice-status').text(
-    sprintf(
-      __('Connected (unencrypted) to: %s'),
-      $('#spice-status').attr('data-host')
-    )
+    sprintf(__('Connected to: %s'), $('#spice-status').attr('data-host'))
```

Comment:
Note for the reviewer: is it worth it to print if it's encrypted or not?

Reply:
Can't hurt.

Reply:
Being able to find that out would be nice, but I guess we can do that during rewrite to a React, that is inevitable.

Reply:
For what it's worth: in practice it'll mostly be encrypted. In production we default to true and you're not allowed to mix HTTP and HTTPS in modern browsers. So I'm guessing it's really a very small number of users who will see unencrypted connections.